zerver/lib/bulk_create.py

```diff
@@ -40,8 +49,13 @@ def bulk_create_users(
             tos_version,
             timezone,
             tutorial_status=UserProfile.TUTORIAL_FINISHED,
-            enter_sends=True,
         )
+        if bot_type is None:
+            for settings_name in RealmUserDefault.property_types:
+                if settings_name in ["default_language", "enable_login_emails"]:
+                    continue
+                value = getattr(realm_user_default, settings_name)
+                setattr(profile, settings_name, value)
```

Comment:
Have not used `copy_default_settings` here because it calls `.save()` and we instead do `bulk_create`. Other way was to pass a variable and do an early return before `.save()` in `copy_default_settings` but I thought this would be cleaner.

Reply:
OK, that's reasonable -- let's just add a comment explaining that ideally we'd use `copy_default_settings`, but we want to avoid the save that it does.

Reply:
I adjusted this comment before merging, thanks!